tests/Makefile

```diff
@@ -11,7 +11,20 @@ include $(TESTSROOTDIR)/Makefile.common
 
 
 # List of disabled tests
-export DISABLED_TESTS:=$(shell awk '/^DISABLED/{f=1;next};/<END>/{f=0}f' TODO | cut -d' ' -f1)
+DISABLED_TESTS:=$(shell awk '/^DISABLED/{f=1;next};/<END>/{f=0}f' TODO | cut -d' ' -f1)
+
+ifeq ($(SKIPSSL), 1)
+SKIPS:=$(shell grep ssl *.test/lrl*  | grep 'ssl_client_mode VERIFY' | cut -f1 -d'/' | sort -u)
```

Comment:
Just grep for 'ssl_client_mode'? presence of this tunable implies an ssl test.

Reply:
many tests have 'ssl_client_mode OPTIONAL' so that wont work.

Reply:
What's OPTIONAL mode? I don't see it in our docs: https://bloomberg.github.io/comdb2/ssl.html. 

Anyway, what about tests with other modes like REQUIRE, VERIFY_HOSTNAME? They won't get skipped with this logic then.

Reply:
IIRC, optional is the mode that if ssl cert exists, it will be used, otherwise we will fallback to non ssl; i might missrememer.
We don't currently have any tests with REQUIRE, and verify_hostname starts with verify so it is correctly skipped.